src/interactive/azext_interactive/azclishell/scenario_suggest.py

```diff
@@ -10,11 +10,23 @@
 
 class ScenarioAutoSuggest(AutoSuggest):
     """Auto Suggest used in Scenario execution mode"""
+
     def __init__(self):
         self.cur_sample = ''
         self.cur_command = ''
+        # param_value_map is the dictionary that has `key` with command params and `value` with <sample_value>
         self.param_value_map = {}
```

Comment:
Could we consider renaming it to `param_sample_value_map`?

Reply:
Received, already modified in https://github.com/wu5bocheng/azure-cli-extensions/commit/f92b35511e194d8a6630747c490e364c43b2039a